### PR TITLE
Fix/similar store names

### LINKED ISSUE #149: persistState - stores with names alike are getting mismatched
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x ] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request
[ ] Other... Please describe
</code></pre>

## Current behavior
If you have 2 stores containing similar strings, they get mismatched on peristState initialization.
e.g.:

1. 'MyStore'
2. 'MyStoreUi'

It fails here:

```
const path = include.find(name => name.indexOf(currentStoreName) > -1);
 if (hasExclude && exclude.indexOf(currentStoreName) > -1 === true) {..,}
```

## Expected behavior
Names should be matched uniquely by name and not by containing string

## Environment

<pre><code>
Angular: 7.1.4
Akita: 1.23.3

### PR DESCRIPTION
**Pull request by twittwer**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/datorama/akita/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
~⚠️ 1 existing test is still crashing (`Persist state - nested key › should persist only the nested key`)~
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: Fixes #149 

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by twittwer** [comment]
@NetanelBasal I have a problem with one test (`Persist state - nested key › should persist only the nested key`)
Can you give me a hint were the problem is coming from?

File: akita/src/enhancers/persist-state.ts [hunk 1]
```diff
@@ -43,6 +43,12 @@ export function persistState(params?: Partial<PersistStateParams>) {
   const hasInclude = include.length > 0;
   const hasExclude = exclude.length > 0;
 
+  const includeStores = {};
```
Comment (NetanelBasal) [inline akita/src/enhancers/persist-state.ts]: Why we need it here?

**Comment by NetanelBasal** [comment]
@twittwer Yes, just move it to a new file, and it should work.

**Comment by NetanelBasal** [comment]
Also, Please add your name to the contributors' list.

**Comment by NetanelBasal** [comment]
Thanks 👍

**Pull request merged**